src/sentry/tasks/post_process.py

```diff
@@ -740,3 +751,21 @@ def plugin_post_process_group(plugin_slug, event, **kwargs):
         _with_transaction=False,
         **kwargs,
     )
+
+
+GROUP_CATEGORY_POST_PROCESS_PIPELINE = {
+    GroupCategory.ERROR: [
```

Comment:
Is there no post process pipeline for transactions? Or is that a totally separate pipeline?

Reply:
It'll be here, but we can include it in a follow up pr.